tests/js/web/PulsarFormComponentTest.js

```diff
@@ -219,4 +229,16 @@ describe('Pulsar Form Component - Select2 elements', function() {
 
     });
 
+    describe('Timepickers', function() {
+
+        beforeEach(function() {
+            this.pulsarForm.timePickerComponent.bindTimePicker = sinon.stub();
+            this.pulsarForm.init();
+        });
+
+        it('should call the timePickerComponents bindTimePicker method', function() {
+            expect(this.pulsarForm.timePickerComponent.bindTimePicker).to.have.been.called;
```

Comment:
Could we go one further and assert `bindTimePicker` is `calledWith(sinon.match.jQuery(...))` to ensure our `[data-timepicker="true"]` elements are passed through to the `timePickerComponent`